.github/workflows/backend-unit-test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: upload coverage profile
         working-directory: ./api
         run: |
-          bash <(curl -s https://codecov.io/bash)
+          bash <(curl -s https://codecov.io/bash) -F backend-unit-test
```

Comment:
Only want to know why this line doesn't need "-f" option.

Reply:
ping @nic-chen

Reply:
@LiteSun  we could use `-F backend` for all backend test

Reply:
I think the back-end unit test and e2e test coverage should be separated, if you want to see the overall back-end coverage, please see https://app.codecov.io/gh/apache/apisix-dashboard/compare/1529/tree.
![image](https://user-images.githubusercontent.com/31329157/109583387-ccd89e00-7b3a-11eb-9d58-71e1c25f2f0b.png)

 What do you think? @membphis @juzhiyuan